canonical-web-and-design/jaas-dashboard
# Use maas ui table components

## Done

Use maas ui table components

## QA

- Pull code
- Run `./run clean && ./run serve`
- Open http://0.0.0.0:8036/
- Sanity check all the things

## Details

Fixes https://github.com/canonical-web-and-design/juju-squad/issues/844
